css/style.css

```diff
@@ -2,19 +2,15 @@ body, html {
     margin: 0;
     padding: 0;
 }
-.container{
-    width: 80%;
-    margin-left: 10%;
+.lb-container{
+    padding: 15px;
+    background-color: rgba(255, 255, 255, .2);
 }
-#list li{
-    list-style: none;
-    margin: 0 auto;
+#images img{
+    margin-top: 15px;
 }
-#list img{
-    width: 100px;
-    position: relative;
-    display: inline-block;
-}
-.imgArray{
+.col-xs-6, .col-md-3, .col-lg-1{
+    float: none;
     display: inline-block;
+    vertical-align: middle;
```

Comment:
The custom class name can simply replace line 12 `.col-xs-6, .col-md-3, .col-lg-1{`. Then in your HTML, you will add your class to the image wrapping column. See comment on `main.js` for more details.